src/main/java/com/github/vkorobkov/jfixtures/domain/Row.java

```diff
@@ -48,6 +50,22 @@ public Row columns(Object... keyValuePairs) {
         return columns(keyValueMap);
     }
 
+    public Row column(String name, Object value) {
+        return columns(singletonMap(name, value));
+    }
+
+    public Row nullColumn(String name) {
+        return columns(singletonMap(name, Value.ofNull()));
```

Comment:
It maybe cleaner if `nullColumn`, `sqlColumn` and `textColumn` methods use `column` method internally - that avoid the singleton map definition:
```java
public Row nullColumn(String name) {
  return column(name, Value.ofNull());
}
```
What do you think?

Reply:
:+1: